op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -147,6 +147,9 @@ func setupLoadTest(t devtest.T, ctx context.Context, wg *sync.WaitGroup) (*AIMD,
 		eoas:         NewEOAPool(dsl.NewFunder(sys.Wallet, sys.FaucetB, l2ELB), budget),
 		el:           l2ELB,
 	}
+	info, err := dest.el.Escape().EthClient().InfoByLabel(ctx, eth.Unsafe)
+	t.Require().NoError(err)
+	dest.unsafe.Store(info)
```

Comment:
Fixing a pre-existing race condition that is apparently easier to hit with the change mentioned in the other comment.